UITests/AuthenticationManagerTests.swift

```diff
@@ -148,14 +122,11 @@ class AuthenticationManagerTests: KIFTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
-        setPasscode("1337", interval: .Immediately)
+        PasscodeUtils.setPasscode("1337", interval: .Immediately)
 
         openAuthenticationManager()
         tester().tapViewWithAccessibilityLabel("Require Passcode, Immediately")
 
-        tester().waitForViewWithAccessibilityLabel("Enter Passcode")
-        enterPasscodeWithDigits("1337")
-
```

Comment:
@sleroux : yes, there is a bug in menu whereby we are not checking that the number of menu items is 0 before we try and use it in a division, but the main problem seems to be that the test is trying to perform actions before a page has loaded. In fact, it seems as if the page never actually loads, so we are in a Loading state rather than on a webpage or a home panel. I think that needs to be addressed - I'm sure it never used to behave like that on these tests.